src/main/java/org/sagebionetworks/bridge/dao/SurveyDaoSQL.java

```diff
@@ -0,0 +1,100 @@
+package org.sagebionetworks.bridge.dao;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+import org.sagebionetworks.bridge.models.GuidCreatedOnVersionHolder;
+import org.sagebionetworks.bridge.models.PagedResourceList;
+import org.sagebionetworks.bridge.models.studies.StudyIdentifier;
+import org.sagebionetworks.bridge.models.surveys.SurveySQL;
+
+public interface SurveyDaoSQL {
```

Comment:
This interface is identical to SurveyDao. Why not use the same interface for both?